.github/workflows/actions/publish-npm.yml

```diff
@@ -17,9 +17,9 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: actions/setup-node@v3
+    - uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # 3.6.0
```

Comment:
Normally, we prefix these little comments with a 'v' (e.g. `# v3.6.0`). 

It's probably not a big deal, I know that Dependabot/Renovate (neither of which are in this repo ATM) are smart enough to bump the version comment as well as the SHA ([like so](https://github.com/ionic-team/stencil/pull/4266/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L14)). No action required here, just wanted to get that thought out for future considerations